packages/geo/tsconfig.lib.json

```diff
@@ -10,5 +10,8 @@
       "@igo2/*": ["dist/*"]
     }
   },
-  "exclude": ["src/test.ts", "**/*.spec.ts"]
+  "exclude": ["src/test.ts", "**/*.spec.ts"],
+  "angularCompilerOptions": {
+    "compilationMode": "full"
```

Comment:
The partial distribution is not possible for now cause of circular references in the template. We could open an issue if we go ahead with this

Reply:
circular references in the template ?